### PR TITLE
docs: Documented round initialization edge-case under the ethereum docs

### PR DESCRIPTION
**Pull request by RiccardoBiosas**
**What does this pull request do? Explain your changes. (required)**
It documents a round-initialization edge-case observed while testing Arbitrum Nitro - described [here](https://github.com/livepeer/internal-project-tracking/issues/383)

**Specific updates (required)**
Updated `doc/ethereum.md`

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**
https://github.com/livepeer/internal-project-tracking/issues/383

**Checklist:**

- [ ] Read the [contribution guide](./doc/contributing.md)
- [ ] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: doc/ethereum.md [hunk 1]
```diff
@@ -45,4 +45,11 @@ The following options can be used to set the min gas price to `<MIN_GAS_PRICE>`,
...
 - Start the node with `-minGasPrice <MIN_GAS_PRICE>`
 - `curl localhost:7935/setMinGasPrice?minGasPrice=<MIN_GAS_PRICE>`
-- Run `livepeer_cli` and select the set min gas price option
\ No newline at end of file
+- Run `livepeer_cli` and select the set min gas price option
+
+### Known edge-cases
+A known edge-case that affects the initialization of new rounds occurs when the L2 block-rate is significantly slower than the L1 block-rate. This may result in:
+- go-livepeer not attempting to initialize a new round based on the expected `roundLength` interval - which is based on L1 block values
+- go-livepeer attempting to redeem a winning ticket after it already expired due to the block-rate difference between L1 and L2
```
Comment (leszko) [inline doc/ethereum.md]: I think that one more result may be the `TicketParams expired` error.

**Review by leszko — APPROVED** [review]
Added one comment. Other than that, LGTM 👍
